fiftyone/migrations/revisions/admin/v0_15_1.py

```
@@ -40,13 +40,20 @@ def _get_database_name():
     return config.get("database_name", "fiftyone")
 
 
-def _load_config():
+def _get_config_path():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_get_config_path`

(ANN202)